samples/CallWithChat/src/app/views/CallScreen.tsx

```diff
@@ -150,9 +150,11 @@ export const CallScreen = (props: CallScreenProps): JSX.Element => {
 
   const options: CallWithChatCompositeOptions = useMemo(
     () => ({
-      callControls: {
-        screenShareButton: !shouldDisableScreenShare
```

Comment:
Is this change necessary anymore if we are dictating never show screenshare on mobile?

Reply:
Yes, I need to hide the screenshare button on IPad. `formFactor` is assigned to desktop on Safari for IPad.

Reply:
Also when I set `formFactor` to mobile for IPad. The call control buttons would be compact.